pallets/loans/src/lib.rs

```diff
@@ -318,6 +320,11 @@ pub mod pallet {
     #[pallet::getter(fn close_factor)]
     pub type CloseFactor<T: Config> = StorageMap<_, Twox64Concat, CurrencyId, Ratio, ValueQuery>;
 
+
+    /// Links a currency id to a market or in other words, dictates the currency of a market.
+    #[pallet::storage]
+    pub type MarketMapper<T: Config> = StorageMap<_, Blake2_128Concat, CurrencyId, Market>;
```

Comment:
blake2 is a secure cryptographic hash function and it recommend by Parity, thus the reason of using `Blake2_128Concat` instead of `Twox64Concat`.

Reply:
if it's CurrencyId you can use `Twox64Concat` as it's controlled by decoding instead of the users

Reply:
Storage is now using `Twox64Concat`